src/atomate2/vasp/flows/core.py

```diff
@@ -171,6 +174,62 @@ def make(self, structure: Structure, prev_dir: str | Path | None = None) -> Flow
         return Flow(jobs, outputs, name=self.name)
 
 
+@dataclass
+class MVLGWBandStructureMaker(Maker):
```

Comment:
I think these files should go in `vasp/flows/mvl.py` since they are using the MVL input sets, rather than the atomate2 ones (which have different settings e.g., for static calculations).